CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Semantic Release Automated Changelog
 
+# [3.1.0-beta.2](https://github.com/AlaskaAirlines/auro-checkbox/compare/v3.1.0-beta.1...v3.1.0-beta.2) (2024-09-16)
+
+
+### Bug Fixes
+
+* **color:** resolve color theming bugs [#128](https://github.com/AlaskaAirlines/auro-checkbox/issues/128) ([15a433e](https://github.com/AlaskaAirlines/auro-checkbox/commit/15a433e376277fcd0b794c59b32ebe2670f6e271))
+
+# [3.1.0-beta.1](https://github.com/AlaskaAirlines/auro-checkbox/compare/v3.0.1...v3.1.0-beta.1) (2024-07-20)
+
+
+### Features
+
+* add tier 3 tokens for theming capabilities [#128](https://github.com/AlaskaAirlines/auro-checkbox/issues/128) ([1a8246e](https://github.com/AlaskaAirlines/auro-checkbox/commit/1a8246e19e1e975d282fb01a81175ce02394e3f8))
+
```

Comment:
We should be removing all "beta" change records prior to merging to main.

Reply:
Issue should be resolved.